qbraid/runtime/origin/provider.py

```
@@ -0,0 +1,153 @@
+    )
+
+
+def _get_service(api_key: str):
+    """Return a QCloudService instance for the given API key."""
+    # pylint: disable-next=import-outside-toplevel
+    from pyqpanda3 import qcloud as qcloud_module
+
+    return qcloud_module.QCloudService(api_key=api_key)
+
+
+def _get_qcloud_job(job_id: str, service: Optional[Any] = None):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_get_qcloud_job`

(ANN202)